src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let discount;
+
+  if (days < 3) {
+    discount = 0;
+  }
+
+  if (days >= 3 && days < 7) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
```

Comment:
Create separate variables for discount, calculate and return result in if statement